css-transitions-2/Overview.bs

```diff
@@ -88,7 +88,9 @@ In CSS Transitions Level 2,
 property values are [=transitionable=]
 <em>unless</em> they have an [=animation type=] that is [=not animatable=].
```

Comment:
I think this paragraph should be rewritten a bit more fully to reflect the new approach.

Reply:
I rewrote the whole thing as per rob's suggestion